css/styles.css

```diff
@@ -0,0 +1,196 @@
+:root {
+  --roboto: 'Roboto', sans-serif;
+  --creteRound: 'Crete Round', serif;
+  --poppins: 'Poppins', sans-serif;
+  --blackOlive: #3c3a39;
+  --eerieBlack: #1c1a19;
+  --safetyOrange: #ff6b00;
+  --oldLace: #fff5e1;
+  --white: #fff;
+}
+
+html {
+  font-size: 62.5%;
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: var(--poppins);
+  color: var(--white);
+  font-size: 1.6rem;
+}
+
+/* Global */
+.container {
+  max-width: 120rem;
+  width: 88%;
+  margin: 0 auto;
+}
+
+h2,
+h3 {
+  font-family: var(--roboto);
+  font-weight: bold;
+}
+
+h1 {
+  font-family: var(--creteRound);
+}
+
+h2 {
+  font-size: 1.6rem;
+}
+
+h3 {
+  font-size: 2.4rem;
+}
+
+a {
+  text-decoration: none;
+  color: var(--white);
+}
+
+/* Utility */
+.flex {
+  display: flex;
+}
```

Comment:
Great work! You did a great job using a **CSS** utility class to prevent repeating your code. It's always great to see developers implementing efficient and scalable solutions in their projects. Keep up the good work! 🙌